src/blob_gc_job.cc

```diff
@@ -197,8 +203,24 @@ Status BlobGCJob::DoRunGC() {
       metrics_.gc_bytes_overwritten += blob_index.blob_handle.size;
       continue;
     }
-
-    last_key_valid = true;
+    last_key_is_fresh = true;
+
+    if (blob_gc_->titan_cf_options().blob_run_mode ==
+        TitanBlobRunMode::kFallback) {
```

Comment:
why not just pause the gc in fallback mode

Reply:
Files will never become obsolete unless going through GC.